Update rate-limits.mdx (#4742)
scaleway/docs-content: pages/generative-apis/reference-content/rate-limits.mdx

## What are the limits?

> Any model served through Scaleway Generative APIs gets limited based by:

Reviewer: ```suggestion
Any model served through Scaleway Generative APIs gets limited based on:
```

- Tokens per minute
- Queries per minute

> Base limits apply if you registered a valid payment method and corresponds to [quotas for your organisation](https://www.scaleway.com/en/docs/organizations-and-projects/additional-content/organization-quotas/). This limits are increased automatically if you also validate your identity. 

Reviewer: ```suggestion
Base limits apply if you registered a valid payment method, and these limits correspond to [quotas for your Organization](organizations-and-projects/additional-content/organization-quotas/). Limits are increased automatically if you also [verify your identity](/account/how-to/verify-identity/). 
```

> Base limits apply if you registered a valid payment method, and these limits correspond to [quotas for your Organization](organizations-and-projects/additional-content/organization-quotas/). Limits are increased automatically if you also [verify your identity](/account/how-to/verify-identity/). 

Reviewer 2: ```suggestion
Base limits apply if you registered a valid payment method, and these limits correspond to [quotas for your Organization](/organizations-and-projects/additional-content/organization-quotas/). Limits are increased automatically if you also [verify your identity](/account/how-to/verify-identity/). 
```
